cli/ballerina-launcher/src/main/java/org/ballerinalang/launcher/Main.java

```diff
@@ -295,16 +293,16 @@ public void setSelfCmdParser(JCommander selfCmdParser) {
      *
      * @since 0.8.0
      */
-    @Parameters(commandNames = "help", commandDescription = "print usage information")
+    @CommandLine.Command(name = "help", description = "Prints usage information")
```

Comment:
The wording has to be from third person context. "print usage information" seems correct for me.

Reply:
yes, will fix all commands.